src/arithmetic/arithmetic_expression.h

```diff
@@ -34,17 +34,19 @@ typedef enum {
 } AR_OPType;
 
 /* AR_OperandNodeType type of leaf node,
- * either a constant: 3, or a variable: node.property. */
+ * either a constant: 3, a variable: node.property, or a parameter. */
 typedef enum {
 	AR_EXP_OP_UNKNOWN,
 	AR_EXP_CONSTANT,
 	AR_EXP_VARIADIC,
+	AR_EXP_PARAM
 } AR_OperandNodeType;
 
 /* Success of an evaluation. */
 typedef enum {
 	EVAL_OK = 0,
 	EVAL_ERR = (1 << 0),
+	EVAL_FOUND_PARAM = (1 << 1),
```

Comment:
Hopefully this addition can be avoided, as mentioned at https://github.com/RedisGraph/RedisGraph/pull/990/files#r384113637